src/main/java/by/exadel/internship/service/impl/SchedulingServiceImpl.java

```diff
@@ -0,0 +1,160 @@
+package by.exadel.internship.service.impl;
+
+import by.exadel.internship.dto.InterviewDTO;
+
+import by.exadel.internship.dto.timeForCall.UserTimeSlotWithUserIdDTO;
```

Comment:
why do you use camel case in pakage name? In line number 10, snake case is used. Choose please single stile.

Reply:
Done